extra_docs/dataset/README.md

```diff
@@ -0,0 +1,269 @@
+# Datasets Module
+A Dataset is used to manipulate records.
+
+### [Initialize](#init) 
+To use dataset features, just add `dataOperations` module to the client initialization. Notice, that for Node.js
+you have to import `dataOperations` from `jexia-sdk-js/node`, but if you are running your application in a browser, you should import it from `jexia-sdk-js/browser`. That is because file format is different under these two platforms.
+```typescript
+import { jexiaClient, dataOperations } from "jexia-sdk-js/node"; // "jexia-sdk-js/browser" for browser applications
+
+const dataModule = dataOperations();
+
+jexiaClient().init(credentials, dataModule);
+```
+
+### [Inserting records](#inserting-records)
+
+Records can be inserted to a dataset either by sending an array or a single one. The response will always be an array though.
+
+``` Javascript
+[..]
+const posts = dataModule.dataset("posts");
+
+// Multiple records
+const insertQuery = posts.insert([
+  { title: "New Post", content: "content here" },
+  { title: "Another Post", content: "some more content" },
+]);
+
+// Single record
+const insertQuery = posts.insert({
+  title: "New Post",
+  content: "content here",
+});
+
+// Either way, the response will be an array
+insertQuery
+  .execute()
+  .then((records) => {
```

Comment:
We are using `async` for the fileset doc, would be nice to have the same approach throughout the whole doc, what do you think?